src/mailadm/cmdline.py

```diff
@@ -58,6 +58,29 @@ def get_mailadm_db(ctx, show=False, fail_missing_config=True):
     return db
 
 
+def create_bot_account(ctx, email: str, password=None) -> (str, str):
+    """Make sure that there is a mailcow account to use for the bot.
+
+    This method tries to use the --email and --password CLI arguments. If they are incomplete
```

Comment:
this sentence abruptly ends.

Reply:
oops. Is it funnier if I say it was intentional? Or if it wasn't?